ludwig/schema/lr_scheduler.py

```diff
@@ -48,6 +30,12 @@ class LRSchedulerConfig(schema_utils.BaseMarshmallowConfig, ABC):
         parameter_metadata=TRAINER_METADATA["learning_rate_scheduler"]["decay_steps"],
     )
 
+    decay_steps: int = schema_utils.PositiveInteger(
```

Comment:
Did you mean to add these parameters as part of this PR?

Reply:
Looks like they are just re-ordered.

Reply:
Obviously... sorry for the brain freeze @connor-mccorm ! Resolving this.